packages/form-core/types/FormControlMixinTypes.d.ts

```diff
@@ -1,5 +1,5 @@
 import { CSSResultArray } from 'lit-element';
-import { LitElement, nothing, TemplateResult } from '@lion/core';
+import { LitElement, nothing, TemplateResult, CSSResultArray } from '@lion/core';
```

Comment:
We would need to re-export this first in `@lion/core`, the problem is that `lit-element` only exports this as a type export. So we would need to re-export this type export (if possible?) in our index.d.ts file in `@lion/core`.

Reply:
Added to '@lion/core'